libs/cart/src/reducers/cart/cart.reducer.ts

```diff
@@ -46,6 +47,13 @@ export function cartReducer<T extends DaffCart>(
         loading: false,
       };
 
+    case DaffCartActionTypes.CartStorageFailureAction:
```

Comment:
you could instead just add line 50 under line 43, because the action should have the error message as a payload

Reply:
Fixed in new commits.